src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+#spring.datasource.url=
```

Comment:
Good 👍 
later in this property file you will be able to add your app.name, port (8080 by default), etc.

Reply:
Thank's.